### PR TITLE
Add last updated to MutinyInvoice

### PR DESCRIPTION
**Pull request by benthecarman**

### CODE REVIEW

**Review by futurepaul — COMMENTED** [review]
why is this optional, wouldn't it be set at creation?

File: mutiny-core/src/nodemanager.rs [hunk 1]
```diff
@@ -117,6 +118,7 @@ impl From<Invoice> for MutinyInvoice {
             paid: false,
             fees_paid: None,
             is_send: false, // todo this could be bad
+            last_updated: None,
```
Comment (TonyGiorgio) [inline mutiny-core/src/nodemanager.rs:121]: Should you use the timestamp field above?

**Comment by benthecarman** [comment]
> why is this optional, wouldn't it be set at creation?

didn't know how to handle from converting from a normal invoice but @TonyGiorgio showed me the way!

**Review by futurepaul — APPROVED** [review]
thank you!

**Review by TonyGiorgio — APPROVED** [review]

**Pull request merged**